### PR TITLE
Adding attributes to facilitate integration testing of HParams

### PR DESCRIPTION
**Pull request by erzel**
@wchargin please take a look.
Thanks!

### CODE REVIEW

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html [hunk 1]
```diff
@@ -557,7 +562,10 @@
                     // after the transition ends.
                         .attr("d", pathDataFn);
                     window.setTimeout(
-                        ()=>foregroundPaths.each(setPathControlPointsFn),
+                        ()=>{
+                          foregroundPaths.each(setPathControlPointsFn),
```
Comment (wchargin) [inline tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: This should be a semicolon, not a comma.

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html [hunk 2]
```diff
@@ -557,7 +562,10 @@
...
+                          this.removeAttribute("is-dragging")
```
Comment (wchargin) [inline tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: This line should also have a semicolon.

Comment (erzel) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: Thanks! Done.

File: tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html (hunk 1 above)
Comment (erzel) [reply tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html]: Thanks. Done!

**Pull request merged**